biz.aQute.bnd.runtime/src/aQute/bnd/runtime/facade/FrameworkFacade.java

```diff
@@ -338,4 +337,11 @@ private Parameters getImports(Bundle b) {
 			.get(Constants.IMPORT_PACKAGE));
 	}
 
+	private List<ServiceReference<?>> asList(ServiceReference<?>[] refs) {
+		if (refs == null)
+			return Collections.EMPTY_LIST;
```

Comment:
Please use `Collections.emptyList()` otherwise the gradle build emits warnings. I had to fix an issue like this already in this project.

Reply:
why is it an issue?